lang/ja/http-statuses.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    '0'            => '不明なエラー',
```

Comment:
```suggestion
    '0' => '不明なエラー',
```
fix tương tự các line và file khác